### PR TITLE
Adding emissions results to UO reports

### PR DESCRIPTION
**Pull request by rawadelkontar**
### Pull Request Description

added emission results to csv reports

### Checklist (Delete lines that don't apply)

- [ ] Unit tests have been added or updated
- [ ] Documentation has been modified appropriately
- [ ] All ci tests pass (green)
- [ ] An [ISSUE](https://github.com/urbanopt/urbanopt-reporting-gem/issues) has been created that this is addressing. Issues will get added to the Change Log when the change_log.rb script is run.
- [ ] This branch is up-to-date with develop

### CODE REVIEW

File: lib/urbanopt/reporting/default_reports/reporting_period.rb [hunk 1]
```diff
@@ -57,14 +57,16 @@ class ReportingPeriod
                       :net_site_energy_kwh, :net_source_energy_kwh, :total_utility_cost_dollar, :net_utility_cost_dollar, :utility_costs_dollar, :electricity_kwh, :natural_gas_kwh, :propane_kwh, :fuel_oil_kwh, :other_fuels_kwh, :district_cooling_kwh,
                       :district_heating_kwh, :water_qbft, :electricity_produced_kwh, :end_uses, :energy_production_kwh, :photovoltaic,
                       :fuel_type, :total_cost_dollar, :usage_cost_dollar, :demand_cost_dollar, :comfort_result, :time_setpoint_not_met_during_occupied_cooling,
-                      :time_setpoint_not_met_during_occupied_heating, :time_setpoint_not_met_during_occupied_hours, :hours_out_of_comfort_bounds_PMV, :hours_out_of_comfort_bounds_PPD #:nodoc:
+                      :time_setpoint_not_met_during_occupied_heating, :time_setpoint_not_met_during_occupied_hours, :hours_out_of_comfort_bounds_PMV, :hours_out_of_comfort_bounds_PPD,
+                      :emissions_kg, :future_annual_emissions_kg, :future_hourly_emissions_kg, :historical_annual_emissions_kg, :historical_hourly_emissions_kg  #:nodoc:
```
Comment (kflemin) [inline lib/urbanopt/reporting/default_reports/reporting_period.rb:61]: other projects (BuildingSync, SEED, BPS efforts in general) are standardizing on this unit for reporting GHG emissions: MtCO2e (metric ton CO2e)

Should we also do that, or keep it in kg?

Comment (rawadelkontar) [reply lib/urbanopt/reporting/default_reports/reporting_period.rb:61]: we can change it later if needed.

File: lib/measures/default_feature_reports/measure.rb [hunk 2]
```diff
@@ -234,13 +234,16 @@ def energyPlusOutputRequests(runner, user_arguments)
                        'District Cooling Inlet Temperature', 'District Cooling Outlet Temperature',
                        'District Heating Hot Water Rate', 'District Heating Mass Flow Rate',
                        'District Heating Inlet Temperature', 'District Heating Outlet Temperature', 'Cooling Coil Total Cooling Rate',
-                       'Heating Coil Heating Rate', 'ExteriorEquipment:Electricity']
+                       'Heating Coil Heating Rate', 'ExteriorEquipment:Electricity', ]
```
Comment (kflemin) [inline lib/measures/default_feature_reports/measure.rb:237]: should this comma be here?

File: lib/urbanopt/reporting/default_reports/schema/scenario_schema.json [hunk 3]
```diff
@@ -978,4 +999,4 @@
 			}
 		}
 	}
-}
+}
```
Comment (kflemin) [inline lib/urbanopt/reporting/default_reports/schema/scenario_schema.json:1002]: add a return at the end of the file

**Review by kflemin — APPROVED** [review]
looks good except for the stray comma and a return at the end of the file. thanks!

**Pull request merged**